guides/doc-Content_Management_Guide/topics/Importing_Content.adoc

```diff
@@ -108,7 +108,7 @@ endif::[]
 You can select either a repository for RPM files (`yum`), Docker images (`docker`), Files (`file`) or other. Note that if the `yum` content type is selected, you can restrict whether the repository is available to a host based on the host's architecture and OS version.
 . Optional: From the *Restrict to Architecture* list, select the architecture. Ensure that *No restriction*, which is the default value, is selected to make the repository available to all hosts regardless of the architecture.
 . Optional: From the *Restrict to OS Version* list, select the OS version. Ensure that *No restriction*, which is the default value, is selected to make the repository available to all hosts regardless of the OS version.
-. In the *URL* field, enter the URL of the external repository to use as a source.
+. In the *URL* field, enter the URL of the external repository to use as a source. There are only three protocols supported: *http://, https://,* and *file://*. If you are using a *file://* repositories, you have to place it under */var/lib/pulp/sync_imports/ directory*.
```

Comment:
I don't have a strong opinion about things in 2.5.

Some suggestions: we generally use in-line code instead of bold text to things like URLs (and protcols) and paths; also one sentence per line; and maybe reword to "{Project} supports three ...".

totally your call to leave as is or make adjustments.

Reply:
Addressed all your suggestions as it falls into standard practice. :+1: